### PR TITLE
[14.0][FIX] delivery_carrier: Do not send a package to schenker

### PR DESCRIPTION
**Pull request by mt-software-de**
if there is no move line without a package

### CODE REVIEW

**Comment by mt-software-de** [comment]
@jbaudoux, @raumschmiede-joshuaL @hildickethan-S73 could you maybe do the review?

**Review by hildickethan — APPROVED** [review]

File: delivery_schenker/models/delivery_carrier.py [hunk 1]
```diff
@@ -419,6 +420,10 @@ def _schenker_shipping_information_without_packages_volume(self, picking):
         )
 
     def _schenker_shipping_information_without_packages(self, picking):
+        if not any(
+            not move_line.result_package_id for move_line in picking.move_line_ids
```
Comment (jbaudoux) [inline delivery_schenker/models/delivery_carrier.py]: not any not = all

to make it easier to read :)

Comment (mt-software-de) [reply delivery_schenker/models/delivery_carrier.py]: oh yes :-)

**Comment by mt-software-de** [comment]
@jbaudoux could you update your review?

**Comment by mt-software-de** [comment]
> @jbaudoux could you update your review?

ping @jbaudoux :-)

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**